src/sptranfv.f

```diff
@@ -1,69 +1,39 @@
 C> @file
+C> @brief Perform a vector spherical transform
+C>
+C> ### Program History Log
+C> Date | Programmer | Comments
+C> -----|------------|---------
+C> 96-02-29 | IREDELL | Initial.
+C> 1998-12-15 | IREDELL | Generic fft used, openmp directives inserted
+C> 2013-01-16 | IREDELL & MIRVIS | Fixing afft negative sharing effect during omp loops
 C>
-C> Perform a vector spherical transform
 C> @author IREDELL @date 96-02-29
```

Comment:
Author/date should go right after brief, right?

Reply:
No I do:

breif

File History

extra doc (if any)

author/date


Author/date is always last.

Reply:
Got it. I thought we were doing separate author+date entries for file and for each subroutine.